### PR TITLE
Hardened ingress-nginx 3.34.0

### PR DESCRIPTION
**Pull request by erikwilson**
For https://github.com/rancher/rke2/issues/659

### CODE REVIEW

File: packages/rke2-ingress-nginx/generated-changes/patch/values.yaml.patch [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
      pullPolicy: IfNotPresent
      # www-data -> uid 101
      runAsUser: 101
-@@ -45,7 +44,7 @@
+@@ -26,7 +24,7 @@
+   existingPsp: ""
+ 
+   # Configures the controller container name
+-  containerName: controller
++  containerName: rke2-ingress-nginx-controller
```
Comment (brandond) [inline packages/rke2-ingress-nginx/generated-changes/patch/values.yaml.patch:24]: this is just the container within the pod; curious why this needed to be changed.

**Review by brandond — APPROVED** [review]

**Review by brandond — APPROVED** [review]
LGTM when CI passes

File: packages/rke2-ingress-nginx/generated-changes/patch/values.yaml.patch (hunk 1 above)
Comment (erikwilson) [reply packages/rke2-ingress-nginx/generated-changes/patch/values.yaml.patch:24]: it is, they changed to `- name: {{ .Values.controller.containerName }}` in ./packages/rke2-ingress-nginx/charts/templates/controller-{daemonset,deployment}.yaml

**Review by briandowns — APPROVED** [review]

File: packages/rke2-ingress-nginx/generated-changes/patch/values.yaml.patch (hunk 1 above)
Comment (erikwilson) [reply packages/rke2-ingress-nginx/generated-changes/patch/values.yaml.patch:24]: I believe it is a leftover from upgrading older versions and the RKE2 CI checks for containers with specific names

**Pull request merged**